Makefile

```diff
@@ -6,6 +6,8 @@ ifeq ($(OS),Windows_NT)
 else
   UNAME_S := $(shell uname -s)
 
+  LINKER_FLAGS = -fuse-ld=gold
```

Comment:
Apparently, the gold linker is the default on MacOS (BTW, I think the CI failed because I didn't add `--export-dynamic-symbol=...` to the link command on MacOS). I'll update that.